overlays/default.nix

```
@@ -1,14 +1,15 @@
 # - Sort packages in alphabetic order.
 # - If the recipe uses `override` or `overrideAttrs`, then use `prev`,
 #   otherwise use `final`.
-
 { inputs }: final: prev:
 let
   utils = import ../shared/utils.nix {
     inherit (final) lib;
   };
 in
 {
+  beautyline-icons = final.callPackage ../pkgs/beautyline-icons { };
```

What about basing its name on the upstream, but adding the `garuda-` prefix? (Also renaming the directory)

```suggestion
  garuda-beauty-line-icon-theme = final.callPackage ../pkgs/garuda-beauty-line-icon-theme { };
```

---

The icon theme is called beautyline rather than beauty-line, the author for some reason decided to rename it. I don't want to spread the incorrect name elsewhere :)